spec/datadog_spec.rb

```diff
@@ -66,16 +66,17 @@
       it 'posts an event' do
         expect(a_request(:post, EVENTS_ENDPOINT).with(
           :query => { 'api_key' => @handler.config[:api_key] },
-          :body => hash_including(:msg_text => 'Chef updated 0 resources out of 0 resources total.'),
-          :body => hash_including(:msg_title => "Chef completed in 5 seconds on #{@node.name} "),
-          :body => hash_including(:tags => ['env:testing']),
+          :body => hash_including(:msg_text => 'Chef updated 0 resources out of 0 resources total.',
```

Comment:
Could you switch to not using rockets for symbols? It will be quick to do and we usually use the `<key>:` syntax for symbol keys.

Reply:
(multiple occurence)